templates/base.tpl

```diff
@@ -1,16 +1,14 @@
 <!doctype html>
```

Comment:
Can we convert all templates into soy templates? Now that we finally have soy support we should discourage the usage of jinja.